algotaurus.py

```diff
@@ -484,6 +508,17 @@ def __init__(self, size=15, lines=30):
         self.editmenu.add_command(label=_('Paste'), command=self.paste_command, accelerator='Ctrl+V')
         self.editmenu.add_separator()
         self.editmenu.add_command(label=_('Select All'), command=self.sel_all, accelerator='Ctrl+A')
+        
+        self.prefsmenu = tk.Menu(self.menu, tearoff=False)
+        self.menu.add_cascade(label=_('Preferences'), menu=self.prefsmenu)
+        self.languagemenu = tk.Menu(self.prefsmenu, tearoff=False)
+        self.prefsmenu.add_cascade(label=_('Language'), menu=self.languagemenu)
+        for lang in sorted(languages.keys()):
+            self.languagemenu.add_radiobutton(label=lang, variable = self.lang_value, value=languages[lang],
+                                              command=self.change_language)
+        self.sizemenu = tk.Menu(self.prefsmenu, tearoff=False)
+        self.prefsmenu.add_command(label=_('Maze size'), command=self.size_toolbox)
+        
```

Comment:
Buid preferences menu:
Preferences
-> Languages
--> lang1
--> lang2
--> etc. (based on language dictionary)
-> Maze size -> Opens the toolbox